### DISCUSSION TITLE
Не появляются главы на PC

### DISCUSSION THREAD

**Discussion by Meloni4** [lirix360/ReadmangaGrabber#39] [General]
В 2.0 beta 6 версии всё работает и появляются главы (название, тома и т.д), но при окончании загрузки нигде нету самих глав, хоть процесс кодирования в PDF был и при завершении было написано, что всё скачано. Я указал путь куда именно нужно сохранять и в итоге скачав одну главу там ничего не было. Не понятно куда программа вообще скачивала главы.

**Comment by lirix360** [comment]
Посмотрите в папке Manga в папке с программой.
Баг проверю.

**Reply by Meloni4** [reply]
![Снимок](https://user-images.githubusercontent.com/96995929/147887938-86706aa2-c020-425d-9ec7-f655efb02286.PNG)
![5](https://user-images.githubusercontent.com/96995929/147887993-ace3a41d-2ccb-4e13-8670-2240ec143386.PNG)
Файлы извлечённые из архива. Здесь ничего нет кроме прописанных в блокноте команд.

**Reply by Meloni4** [reply]
После извлечения файлов в отдельную папку всё заработало. Появилась папка Manga и все главы скачиваются.

**Comment by lirix360** [comment]
Ошибка с сохранением в указанное место исправлена в beta 7.
